README.md

```diff
@@ -29,7 +29,7 @@ For a more detailed documentation of the API, please refer to the [API documenta
 Tests that need a running elasticsearch outside of component, e.g. in your local docker are marked with selfdefined java
 annotation test tag  ```IntegrationTag``` in ```/ndla/article-api/src/test/java/no/ndla/tag/IntegrationTest.java```.
 
-As of now we have no running elasticserach or tunnel to one on Travis and need to ignore these tests there or the build will fail.  
+As of now we have no running elasticsearch or tunnel to one on Travis and need to ignore these tests there or the build will fail.  
```

Comment:
Egentlig så kan hele `#### IntegrationTest Tag and sbt run problems` biten slettes siden vi kjører elasticsearch testene i travis nå. (Mot en embedded elasticsearch i koden).